sale_automatic_workflow_job/tests/test_auto_workflow_job.py

```diff
@@ -45,6 +45,7 @@ def test_validate_sale_order(self):
                     ("state", "=", "draft"),
                     ("workflow_process_id", "=", self.sale.workflow_process_id.id),
                 ],
+                False,
```

Comment:
```suggestion
                send_order_confirmation=False,
```

Reply:
I tried but it does not work.

Reply:
@TDu what do you mean?

Reply:
https://github.com/OCA/sale-workflow/blob/4b91bb664d6ca58f7543801bebb8bf753673f1e2/sale_automatic_workflow_job/tests/test_auto_workflow_job.py#L21